### PR TITLE
Beta docs

### PR DESCRIPTION
**Pull request by v-ymundhe**

### CODE REVIEW

**Comment by v-ymundhe** [comment]
@microsoft-github-policy-service agree [company="Microsoft"]

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Add-EntraBetaServicePrincipalOwner.md [hunk 1]
```diff
@@ -9,42 +18,53 @@ schema: 2.0.0
...
-```
-PS C:\> $ServicePrincipalId = (Get-EntraBetaServicePrincipal -Top 1).ObjectId
-PS C:\> $OwnerId = (Get-EntraBetaUser -Top 1).ObjectId
-PS C:\> Add-EntraBetaServicePrincipalOwner -ObjectId $ServicePrincipalId -RefObjectId -$OwnerId
-```
 
-The first command gets the object ID of a service principal by using the Get-EntraBetaServicePrincipal (./Get-EntraBetaServicePrincipal.md)cmdlet, and then stores it in the $ServicePrincipalId variable.
+```powershell
+Connect-Entra -Scopes 'Application.ReadWrite.All' #Delegated Permission
+Connect-Entra -Scopes 'Application.ReadWrite.OwnedBy' #Application Permission
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Add-EntraBetaServicePrincipalOwner.md:43]:
```suggestion
Connect-Entra -Scopes 'Application.ReadWrite.All','Application.ReadWrite.OwnedBy'
```

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaApplicationPassword.md [hunk 2]
```diff
@@ -9,42 +18,60 @@ schema: 2.0.0
...
 ### Example 1: Add a password to an application
+
+```powershell
+Connect-Entra -Scopes 'Application.ReadWrite.All','Directory.ReadWrite.All'
+$PasswordCredential= New-Object Microsoft.Open.MSGraph.Model.PasswordCredential
+$PasswordCredential.StartDateTime = Get-Date -Year 2024 -Month 12 -Day 28
+$PasswordCredential.EndDateTime = Get-Date -Year 2025 -Month 2 -Day 28
+$PasswordCredential.CustomKeyIdentifier = [System.Text.Encoding]::UTF8.GetBytes('a')
+$PasswordCredential.Hint = 'b'
+$PasswordCredential.DisplayName = 'mypassword'
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaApplicationPassword.md:48]: Confirm code logic is already implemented. The main branch does not have this param supported (displayName).

Comment (v-ymundhe) [reply module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaApplicationPassword.md:48]: Updated document according to the new PR
https://github.com/microsoftgraph/entra-powershell/pull/969

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaUserAppRoleAssignment.md [hunk 3]
```diff
@@ -9,69 +19,116 @@ schema: 2.0.0
...
-# Get the service principal for the app you want to assign the user to
+## Examples
 
-$servicePrincipal = Get-EntraBetaServicePrincipal -Filter "appId eq 'appId'"
+### Example 1: Assign a user to an application without roles
 
-# Create the user app role assignment
+```powershell
+Connect-Entra -Scopes 'AppRoleAssignment.ReadWrite.All'
+$appId = (Get-EntraBetaApplication -SearchString 'NewApplication').AppId
```
[the remark this answers is not in the document]
Comment (v-ymundhe) [reply module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaUserAppRoleAssignment.md:54]: Example updated.

File: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaUserAppRoleAssignment.md [hunk 4]
```diff
@@ -9,69 +19,116 @@ schema: 2.0.0
...
+
+- `-ObjectId` parameter specifies the Id of a user to whom you are assigning the app role.
+- `-PrincipalId` parameter specifies the Id of a user to whom you are assigning the app role.
+- `-ResourceId` parameter specifies the Id of a resource servicePrincipal that has defined the app role.
+- `-Id` parameter specifies the Id of a appRole (defined on the resource service principal) to assign to the user.
 
 ### Example 2: Assign a user to a specific role within an application
+
+```powershell
+Connect-Entra -Scopes 'AppRoleAssignment.ReadWrite.All'
```
[the remark this answers is not in the document]
Comment (v-ymundhe) [reply module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaUserAppRoleAssignment.md:85]: Yes example works. Here for Id params user should pass the appropriate role Id. 
image

**Review by SteveMutungi254 — APPROVED** [review]
LGTM.

**Pull request merged**